src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -14,6 +14,18 @@ def _set_container(self, container):
             self._constraints._container = container
         self.rehint()
 
+    def _set_hidden(self, status, children):
+        for view in self._container._impl.subviews:
+            [view.setHidden_(status) for c in children if c._impl == view]
```

Comment:
Yes, is not updating the super view geometry yet, I tryed to use one of the recommendation here "https://stackoverflow.com/questions/19561269/autolayout-with-hidden-uiviews" but is a problem to look more deeply.